### DISCUSSION TITLE
Permission for entering closed islands

### DISCUSSION THREAD

**Discussion by RVSkeLe** [BG-Software-LLC/SuperiorSkyblock2#2070] [Pending Feature Requests]
When a player closes the island, staff members can teleport to them only if they have /is admin bypass, but with that permission, they can also break or place blocks. It would be nice to have a separate permission to enter closed islands.

**Comment by OmerBenGera** [comment]
superior.admin.bypass.close_island
https://wiki.bg-software.com/superiorskyblock/overview/commands-and-permissions/permissions#superior.admin.bypass.less-than-island_privilege-greater-than
